include/crm/pengine/status.h

```diff
@@ -80,6 +67,10 @@ pe_rsc_is_bundled(const pcmk_resource_t *rsc)
     return rsc->variant == pcmk_rsc_variant_bundle;
 }
 
+#if !defined(PCMK_ALLOW_DEPRECATED) || (PCMK_ALLOW_DEPRECATED == 1)
```

Comment:
Seems like this should be in an earlier commit but doesn't matter

Reply:
Ah yes, I shuffled commits around